PublicMapping/districtbuilder-website
# Add Organization links, CTA, and subnav

## Overview
New organizations feature is coming out. This PR adds new navigation items and highlights the feature in a banner & special call to action.

NOTE: THERE ARE CURRENTLY NO REAL LINKS FOR ORGANIZATIONS. It seems likely that this will be a Medium blog post. Links have `NEEDS-LINK` set, so when this page exists it should be easy to update. The call-to-action links to a `mailto:` and will not need to be updated.

### Checklist

- [x] Deploy previews do not contain any content security policy errors

### Demo
New Navigation & banner
<img width="476" alt="Screen Shot 2021-03-19 at 4 44 27 PM" src="https://user-images.githubusercontent.com/5672295/111783459-aef99000-88d3-11eb-93f1-0553378ab9fb.png">
<img width="474" alt="Screen Shot 2021-03-19 at 4 44 36 PM" src="https://user-images.githubusercontent.com/5672295/111783474-b1f48080-88d3-11eb-9bad-5dce146edf4e.png">
<img width="1665" alt="Screen Shot 2021-03-19 at 4 47 40 PM" src="https://user-images.githubusercontent.com/5672295/111783481-b325ad80-88d3-11eb-9fe7-5e90380014e8.png">

New call to action
<img width="475" alt="Screen Shot 2021-03-19 at 4 47 15 PM" src="https://user-images.githubusercontent.com/5672295/111783529-bcaf1580-88d3-11eb-9377-fa0a422d747d.png">
<img width="1666" alt="Screen Shot 2021-03-19 at 4 47 29 PM" src="https://user-images.githubusercontent.com/5672295/111783549-c173c980-88d3-11eb-8333-6f531e2cc863.png">


### Notes
- The footer used to have the exact same navigation as in the header, but with the new subpages I've since split these out.
- Maybe we shouldn't use a bare `mailto:` for the "Request a demo" button? If we shouldn't, we can link this up to the Contact page instead.

## Testing Instructions
- `git pull`
- Note new banner.
- Click "Resources" to see new submenu. Close.
- Scroll till you get to the new call to action. Change screen width to see responsive layouts.
- Test nav and banner for mobile.
- When Organizations page exists, test all links.

Closes https://github.com/azavea/azavea-marketing/issues/310
